native/src/routes/disclaimer/DisclaimerContainer.js

```diff
@@ -102,20 +104,18 @@ class DisclaimerContainer extends React.Component<DisclaimerPropsType, Disclaime
     const { disclaimer, error, timeoutExpired } = this.state
 
     if (error) {
-      return <ScrollView refreshControl={<RefreshControl onRefresh={this.loadDisclaimer} refreshing={false} />}
-                         contentContainerStyle={{ flexGrow: 1 }}>
+      return <LayoutedScrollView refreshControl={<RefreshControl onRefresh={this.loadDisclaimer} refreshing={false} />}>
         <FailureContainer error={error} tryAgain={this.loadDisclaimer} />
-      </ScrollView>
+      </LayoutedScrollView>
```

Comment:
Why is LayoutedScrollView.js used here? before the change there way no LayoutContainer

Reply:
Otherwise the background/surroundings have the wrong color as nothing is rendered there. Did not find a better solution.

Reply:
Wrapping in a `LayoutContainer` was already done implicitly before as well, just not for all components. Not sure why that worked at all.